src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -71,7 +71,22 @@ rendering.macro.status.parameter.colour.description=The color of the status(text
 rendering.macro.status.parameter.title.name=Title
 rendering.macro.status.parameter.title.description=The text of the status. If not specified, the color name will be displayed (e.g.:"Grey").
 rendering.macro.status.parameter.subtle.name=Subtle
-rendering.macro.status.parameter.subtle.description=The aspect of the status: either white text on dark color background or dark color text on light color background. This is a boolean parameter so it should be filled with "true" or "false". Default: "false" (white text on dark color background).</content>
+rendering.macro.status.parameter.subtle.description=The aspect of the status: either white text on dark color background or dark color text on light color background. This is a boolean parameter so it should be filled with "true" or "false". Default: "false" (white text on dark color background).
+
+## Team macro
+rendering.macro.team.name=Team
+rendering.macro.team.name=description=Display avatars of the members of a team
+rendering.macro.team.parameter.tag.description=Tag that users need to have to be shown
```

Comment:
The parameter name also needs to be translated so it also needs to appear in the translation bundle.

Reply:
It seems done for the panel macro but not for the status macro, from which I took example of. Should we fix this?

Reply:
Yes, but with a separate PR.

Reply:
https://github.com/xwikisas/xwiki-pro-macros/issues/65